slices/python3.8-minimal.yaml

```diff
@@ -0,0 +1,13 @@
+package: python3.8-minimal
+
+slices:
+  core:
```

Comment:
I'm not sure this is the right name. we've previously used `core` for scenarios where you can extract a subset from a package that just "works" on its own. here, it is just a binary and everything else comes from the dependencies. To play the devil's advocate: _why would this be `core` while in openssl it is called `bins`?_

Reply:
A'ight, sounds good. I will rename it to `bins`.